core/src/Streamly/Internal/Data/Stream/StreamK/Type.hs

```diff
@@ -15,8 +15,8 @@
 module Streamly.Internal.Data.Stream.StreamK.Type
     (
     -- * StreamK type
-      Stream (..)  -- XXX stop exporting this
-    , StreamK
+    --  Stream (..)  -- XXX stop exporting this
```

Comment:
We can continue exporting this in case anyone is using it from Internal modules. But we can deprecate it.